tools/WebhookProcessor/secret.php

```diff
@@ -0,0 +1,31 @@
+<?php
+//This file contains things that should not be touched by the automatic live tracker
+ 
+//Github lets you have it sign the message with a secret that you can validate. This prevents people from faking events.
+//This var should match the secret you configured for this webhook on github.
+//This is required as otherwise somebody could trick the script into leaking the api key.
+
+global $hookSecret;
+$hookSecret = '08ajh0qj93209qj90jfq932j32r';

```

Comment:
It worked perfectly fine without the globals in this file on my test.

I was not able to see anything that would explain this being required skimming thru the 5->7 compilability changes list